services/venice-controller/src/main/java/com/linkedin/venice/controller/server/CreateVersion.java

```diff
@@ -781,9 +782,19 @@ public Route emptyPush(Admin admin) {
         String pushJobId = request.queryParams(PUSH_JOB_ID);
         int partitionNum = admin.calculateNumberOfPartitions(clusterName, storeName);
         int replicationFactor = admin.getReplicationFactor(clusterName, storeName);
+
+        Store store = admin.getStore(clusterName, storeName);
+        if (store == null) {
```

Comment:
should we retry ?

Reply:
I think it's ok to not retry since this is just empty push which is mostly invoked via CLI and can be retried from user's end